src/app/app.component.spec.ts

```diff
@@ -20,16 +20,16 @@ describe('AppComponent', () => {
     expect(app).toBeTruthy();
   });
 
-  it(`should have as title 'sw'`, () => {
+  it(`should have as title 'stopwatch'`, () => {
     const fixture = TestBed.createComponent(AppComponent);
     const app = fixture.componentInstance;
-    expect(app.title).toEqual('sw');
+    expect(app.title).toEqual('stopwatch');
   });
 
   it('should render title', () => {
     const fixture = TestBed.createComponent(AppComponent);
     fixture.detectChanges();
     const compiled = fixture.nativeElement as HTMLElement;
-    expect(compiled.querySelector('.content span')?.textContent).toContain('sw app is running!');
+    expect(compiled.querySelector('.content span')?.textContent).toContain('stopwatch app is running!');
```

Comment:
this won't work (you don't have this text on the page), so let's delete tests which will fail for now